src/administration/integrations.md

```diff
@@ -12,5 +12,4 @@ platform integrations
 ![Cli Integrations](/images/cli-integrations.png)
 
 > **note**
-> If you have created your account using the bitbucket or github oAuth Login in order to use the Platform CLI you will need to setup a
-> password which you can do by visiting this page [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password)
+> If you have created your account using the Bitbucket or GitHub oAuth Login, then in order to use the Platform CLI, you will need to set up a password by visiting [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password).
```

Comment:
One of the things I hate most about the English language is how subjective it is in regards to "optional" grammar.  The way I was taught how to know whether to include a comma in a sentence or not was if it separates a topic or complete thought.  If the sentence still makes sense when you take out `then in order to use the Platform CLI`, then the comma is necessary.  In this case, the sentence still makes sense.  I can make the change you suggest but my English teacher would not approve. :)